docs/code_ref/net.rst

```diff
@@ -34,9 +34,6 @@ VSO
 .. automodapi:: sunpy.net.vso
    :headings: ^#
 
-.. automodapi:: sunpy.net.vso.vso
```

Comment:
This is the missing bit of the docs I added back in recently! Is there a reason it's going again?

Reply:
(xref #2353)

Reply:
Yeah, sorry about that. We had some of the vso API already imported in the vso/__init__ file so we had a duplicate in the docs search. I added the vso ones into the __init__ so it should be there now. I might have overlooked some of the other ones. I will check.

Reply:
@dstansby can you remember the objects that were missing? as far as I could tell it was only `QueryResponse` which this PR moves to the `sunpy.net.vso` namespace.

Reply:
It was `QueryResponse` and `VSOClient` - which appear in the devdocs here: http://docs.sunpy.org/en/latest/code_ref/net.html#module-sunpy.net.vso.vso (but I guess `VSOClient` was duplicated)

(I'm also very confused by documentation for code that lives in `net.vso.vso` ending up under in `net.vso` package documentation)

Reply:
You mean that to use the functions under `net.vso.vso`, you can import them under the `net.vso` namespace?

Reply:
Ah, so if I import `net.vso` I get the stuff in `net.vso.vso` too. Got it!

Reply:
You get everything defined in `net/VSO/__init__.py` which with this PR incluses `QueryResponse` and `VSOClient` imported from `vso.vso`. Which now means both of those are documented as members of `sunpy.net.vso`